### PR TITLE
Fixes in ContainerManager. Volumes parameter, wait parameter, CS fixes

### PR DESCRIPTION
**Pull request by r0b1n**
Small fixes in `client` methods invocation.
- `wait` parameter fix. It should be passed to options, not to query.
- `volumes` (`v`) mistake fix.
- CS fixes.

### CODE REVIEW

**Comment by hacfi** [comment]
:+1: LGTM

File: src/Docker/Manager/ContainerManager.php [hunk 1]
```diff
@@ -399,8 +408,9 @@ public function stop(Container $container, $timeout = 5)
         $response = $this->client->post(['/containers/{id}/stop?t={timeout}', [
             'id' => $container->getId(),
             'timeout' => $timeout,
+        ]],[
             'wait' => true,
-        ]]);
+        ]);
```
Comment (hacfi) [inline src/Docker/Manager/ContainerManager.php]: :+1: just noticed this as well

File: src/Docker/Manager/ContainerManager.php [hunk 2]
```diff
@@ -425,9 +435,11 @@ public function remove(Container $container, $volumes = false)
     {
         $response = $this->client->delete(['/containers/{id}?v={volumes}', [
             'id' => $container->getId(),
-            'v' => $volumes,
+            'volumes' => $volumes ? 1 : 0,
```
Comment (ubermuda) [inline src/Docker/Manager/ContainerManager.php]: As discussed in #103 can you use type casting here?

**Comment by r0b1n** [comment]
PR merged with upsteam/master. It shows 4 commits not..  Guys, what I did wrong?

**Comment by joelwurtz** [comment]
You need to rebase, not merge

git remote update,
git rebase upstream/master

**Comment by r0b1n** [comment]
Thanks :beers:

**Comment by joelwurtz** [comment]
Sorry for the wait  thanks for the PR :beers: :)

**Pull request merged**